src/slime_parser.peg.eex

```diff
@@ -91,13 +87,10 @@ text_block <- text_block_line (crlf
 text_block_nested_lines <- text_block_line (crlf (
   indent lines:text_block_nested_lines dedent / lines:text_block_nested_lines
 ))*;
-text_block_line <- space? (dynamic:text_with_interpolation / simple:text);
+text_block_line <- space? text_item*;
```

Comment:
`text_item` only matches a non-empty string, which works well for e.g. `inline_html` where text is mandatory, but needs the optional modifier for empty string consumers like `text_block_line`.